aap-common/ref-aap-using-playbooks.adoc

```diff
@@ -46,72 +46,35 @@ localhost                  : ok=29   changed=1    unreachable=0    failed=0    s
 A _failed_ not equal zero indicates an issue with Ansible on Cloud deployment.
 
 [discrete]
-== gcp_add_labels
+== gcp_manage_labels
 
-This playbook adds labels to the deployment.
+This playbook adds/removes labels to the deployment.
 
 [literal, options="nowrap" subs="+attributes"]
 ----
-$ docker run --rm $IMAGE command_generator_vars gcp_add_labels
+$ docker run --rm $IMAGE command_generator_vars gcp_manage_labels
 ----
 
 Which generates the following output:
 
 [literal, options="nowrap" subs="+attributes"]
 ----
 ===============================================
-Playbook: gcp_add_labels
-Description: This playbook adds labels to the deployment.
+Playbook: gcp_manage_labels
+Description: This playbook adds/removes labels to the deployment.
 -----------------------------------------------
-Add labels to the Ansible Automation Platform from GCP Marketplace deployment.
+Add/Remove labels to the Ansible Automation Platform from GCP Marketplace deployment.
```

Comment:
Add labels to or remove labels from the {AAPonGCP} deployment

Reply:
fixed